docs/resources/aws_security_group.md.erb

```diff
@@ -260,6 +283,7 @@ The matchers accept a key-value list of search criteria.  For a rule to match, i
   * position - A one-based index into the list of rules. If provided, this restricts the evaluation to the rule at that position. You may also use the special values `:first` and `:last`. `position` may also be used to enable `allow_in_only` and `allow_out_only` to work with multi-rule Security Groups.
   * protocol - Specifies the IP protocol. 'tcp', 'udp', and 'icmp' are some typical values. The string "-1" or 'any' is used to indicate any protocol.
   * to_port - Determines if a rule exists whose port range ends at the specified number. The word 'to_' does *not* relate to inbound/outbound directionality; it relates to the port range ("counting _to_"). `to_port` is an exact criterion; so if the rule allows 1000-2000 and you specify a `to_port` of 1999, it does not match.
+  * security_group - Specifies a security-group id, to be checked as permissible origin (for `allow_in`) or destination (for `allow_out`) for traffic. Each AWS Security Group rule may have multiple allowed source or destination security groups.
```

Comment:
Well, I'm not sure. Maybe I should just remove that sentence. The thing is: You can't actually create a rule with multiple security groups as source/destination in the AWS Web interface. What happens is that if you create multiple identical rules which only differ on the security group they refer to, the API returns a struct containing one rule definition with multiple user_id_group_pairs in an array as targets/sources. I don't expect users to actually want to test for a list of security groups allowed for a single rule. This would be counterintuitive as it's not the way rules are created. Users will expect there to be separate rules while the API returns one rule with, that's why I implemented it to work with single sg's only

Reply:
Understood - that sounds like an edge case.  I'm happy with just documenting its current behavior.